src/shared/utils/convertUserInput.ts

```diff
@@ -1,5 +1,8 @@
 import React from 'react'
 
+export const convertUserInputValueToNumOrNaN = (value?: any): number =>
+  value === '' ? NaN : Number(value)
```

Comment:
Why does `''` get converted to `NaN` and not `Number('') // 0`?

Reply:
This is specifically used when we have user input. An empty string means the user has deleted all of the input field. We should interpret that case as "not a number" rather than 0.